packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js

```diff
@@ -204,6 +206,7 @@ PrayerCard.propTypes = {
   advancePrayer: PropTypes.func,
   incrementPrayer: PropTypes.func,
   prayerId: PropTypes.string,
+  prayerRequest: PropTypes.shape({}),
```

Comment:
```suggestion
```

Reply:
This is still needed because of the `props.prayerRequest.id`